app/serializers/pull_request_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class PullRequestSerializer < ActiveModel::Serializer
-  attributes :id, :title, :repository_name, :owner_id, :html_url, :likes, :created_at, :owner_name
+  attributes :id, :title, :repository_name, :owner_id, :html_url, :likes, :created_at, :owner_name, :description
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/LineLength: Line is too long. [112/100]